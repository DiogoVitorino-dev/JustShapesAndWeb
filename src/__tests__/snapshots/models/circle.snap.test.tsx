import { render, renderHook } from "@testing-library/react-native";
import { useSharedValue } from "react-native-reanimated";

import Circle, { CircleData } from "@/models/geometric/circle";

describe("Circle model - Snapshot test", () => {
  it("Should renders correctly", () => {
    const data = renderHook(() => useSharedValue<CircleData>({ x: 25, y: 50 }));

    const tree = render(<Circle data={data.result.current} />);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
