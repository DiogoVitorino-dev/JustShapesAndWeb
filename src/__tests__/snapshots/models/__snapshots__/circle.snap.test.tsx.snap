// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Circle model - Snapshot test Should renders correctly 1`] = `
<View
  animatedStyle={
    {
      "value": {
        "borderRadius": 25,
        "height": 50,
        "left": 25,
        "top": 50,
        "width": 50,
      },
    }
  }
  collapsable={false}
  style={
    {
      "backgroundColor": "rgb(255, 64, 20)",
      "borderRadius": 25,
      "height": 50,
      "left": 25,
      "position": "absolute",
      "top": 50,
      "width": 50,
    }
  }
  testID="circleModel"
/>
`;
